### ISSUE TITLE
[New Rule] write input() instead of input("")

### ISSUE THREAD

**Issue by MartinThoma** [MartinThoma/flake8-simplify#135]
## Explanation

`input()` does the same as `input("")`, but has less clutter

## Example

```python
# Bad
input("")

# Good
input()
```
